src/components/UI/AlertBox/MyErrorBox.jsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react'
-import classes from './MyErrorBox.module.css'
+import React from "react";
+import classes from "./MyErrorBox.module.css";
 
 const MyErrorBox = () => {
   return (
     <div className={classes.Box}>
-      <h1 style={{ fontSize: '20px' }}>Неверное название города</h1>
+      <h1 style={{ fontSize: "20px" }}>Error</h1>
```

Comment:
It's bad practice to mix code and styles.
If you need to change the visual look dynamically you should use classes or attributes and change them.

Reply:
added temporary solution